storage/Action.cc

```diff
@@ -95,11 +99,17 @@ namespace storage
 			sid_t child_sid = actiongraph.get_devicegraph(LHS)->get_impl()[*vi2]->get_sid();
 
 			vector<Actiongraph::Impl::vertex_descriptor> tmp = actiongraph.actions_with_sid(child_sid, ONLY_LAST);
-			if (!tmp.empty())
-			    actiongraph.add_edge(tmp.front(), v);
+			if (!tmp.empty()) {
```

Comment:
Please follow the same coding style as in the rest of that library: Opening curly braces go to the next line.